conferenceOwen/src/screens/menuUser/UpcomingScreen.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {SafeAreaView, Text, StyleSheet} from 'react-native';
```

Comment:
Try to keep one new line in each import. if you could do it in every file it would be great like this:

```
import {
  SafeAreaView,
  Text,
  StyleSheet
} from 'react-native';
```